packages/os/os.spec

```
@@ -104,7 +104,7 @@ Requires: (%{_cross_os}cfsignal if %{_cross_os}variant-platform(aws))
 
 Requires: (%{_cross_os}warm-pool-wait if %{_cross_os}variant-family(aws-k8s))
 
-Requires: (%{_cross_os}driverdog if %{_cross_os}variant-flavor(nvidia))
+Requires: (%{_cross_os}driverdog if %{_cross_os}variant-flavor(nvidia) or %{_cross_os}variant-flavor(nvidia-fips))
```

Seems like we can just relax this and include everywhere:
```suggestion
Requires: (%{_cross_os}driverdog
```

---

I have a [draft change](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/644/files) that was moving this up to the sorted list on line 81 and removing the constraint for NVIDIA variants. This would be fine to take by itself right now.

---

Ok, done.